### ISSUE TITLE
makeTxDbFromUCSC fails for hg38

### ISSUE THREAD

**Issue by Austin-s-h** [Bioconductor/GenomicFeatures#22]
Hey, I was trying to use the command makeTxDbFromUCSC() on hg38 and it seems to be failing. I'm trying to retrieve the knownGenes. A little investigation, supportedUCSCtables() returns "knownGeneOld8", but even specifying that in the tablename argument fails. 

```
TxDb_hg38 <- makeTxDbFromUCSC(genome = "hg38", tablename = "knownGene")
Error in .tablename2track(tablename, session) : 
  UCSC table "knownGene" is not supported
```
```
supportedUCSCtables(genome = "hg38")
               tablename          track          subtrack
1          knownGeneOld8 Old UCSC Genes              <NA>
2          knownGeneOld7 Old UCSC Genes              <NA>
3          knownGeneOld6 Old UCSC Genes              <NA>
4          knownGeneOld4 Old UCSC Genes              <NA>
5          knownGeneOld3 Old UCSC Genes              <NA>
6               ccdsGene           CCDS              <NA>
7                refGene    NCBI RefSeq       UCSC RefSeq
8            xenoRefGene   Other RefSeq              <NA>
9                sibGene      SIB Genes              <NA>
10               sgpGene      SGP Genes              <NA>
11                geneid   Geneid Genes              <NA>
12               genscan  Genscan Genes              <NA>
13            ncbiRefSeq    NCBI RefSeq        RefSeq All
14     ncbiRefSeqCurated    NCBI RefSeq    RefSeq Curated
15       ncbiRefSeqOther    NCBI RefSeq      RefSeq Other
16         ncbiRefSeqPsl    NCBI RefSeq RefSeq Alignments
17   ncbiRefSeqPredicted    NCBI RefSeq  RefSeq Predicted
18 ncbiRefSeqGenomicDiff    NCBI RefSeq      RefSeq Diffs
```
```
TxDb_hg38 <- makeTxDbFromUCSC(genome = "hg38", tablename = "knownGeneOld8")
Error in .tablename2track(tablename, session) : 
  UCSC table "knownGeneOld8" does not exist for genome "hg38", sorry
```

**Comment by hpages** [comment]
Thanks @Austin-s-h for the report. A fix is on its way.

In the mean time please use the TxDb.Hsapiens.UCSC.hg38.knownGene package. The data provided by the UCSC folks in their hg38/knownGene table has not changed since Oct 2019 so TxDb.Hsapiens.UCSC.hg38.knownGene still contains the most current data, that is, it's the same as what `makeTxDbFromUCSC(genome="hg38", tablename="knownGene")` would give you.

**Comment by Austin-s-h** [comment]
That's what I've been using. Thanks!

**Comment by hpages** [comment]
This should be fixed in GenomicFeatures 1.41.2 (see commit 50fbb2f5ec1c378dac3da7726a24e2346f6d34e8).

I'll apply the fix to the RELEASE_3_11 branch in the next few days.

H.

**Comment by hpages** [comment]
Hi @Austin-s-h ,

The fix is now in release too (GenomicFeatures 1.40.1). Update with `BiocManager::install()`.

Cheers,
H.
